extensions/sld/include/powsybl/iidm/extensions/sld/ConnectablePosition.hpp

```diff
@@ -90,6 +98,8 @@ class ConnectablePosition : public Extension {
     OptionalFeeder m_feeder3;
 };
 
+std::ostream& operator<<(std::ostream& stream, const ConnectablePosition::Direction& value);
```

Comment:
To be check: is it really necessary? In `include/powsybl/iidm/Enum.hpp` we have an implementation that should work with all enums. You just have to specialize the `toString()` template method

Reply:
Yes it looks it is necessary unless we cannot write (ConnectablePositionTest.cpp:80)
```
    BOOST_CHECK_EQUAL(ConnectablePosition::Direction::BOTTOM, extension.getFeeder().get().getDirection());
```

It looks like this operator should be defined in the namespace of the class to be ok, which is not the case of the generic operator defined in Enum.hpp (i.e defined in `powsybl::iidm`)